.github/workflows/api-tests-with-private-packages.yml

```diff
@@ -42,7 +43,42 @@ jobs:
 
       - name: Install Dependencies
         if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
-        run: make install-packages
+        shell: bash
+        run: |
+          echo "https://${{ secrets.GH_PRIVATE_ACCESS_TOKEN }}:@github.com" > ${HOME}/.git-credentials
+          git config --global credential.helper store
+          make install-packages opts="--with saml,auth-controller,workflows"
+          rm -rf ${HOME}/.git-credentials
+
+      - name: Install SAML Dependencies
+        run: sudo apt-get install -y xmlsec1
+
+      - name: Get workflows-logic package version
+        id: workflow-version
+        run: |
+          tag=$(make get-workflow-version)
```

Comment:
I think it's better to put all of this, not just version retrieval, into a Makefile target so we're able to run `make test-private` locally. This way we won't have to specify the non-pip dependency versions twice, too.

Reply:
Yep, good shout!  this is done